components/tidb_query_executors/src/util/mod.rs

```diff
@@ -1,15 +1,16 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+use tidb_query_common::Result;
```

Comment:
Unnecessary moves. And it's more common to put `use` below `mod` (though it's not mandatory in TiKV now)

Reply:
addressed. It was auto-formatted by my Clion. I have disabled it and replaced it with rustfmt. 😈